src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+import { useContext, useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleList } from '../PeopleList/PeopleList';
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
```

Comment:
Such imports look bit ugly

Why not rename `PeopleFilters/PeopleFilters.tsx` to just `PeopleFilters/index.tsx`

Than you will get :

```suggestion
import { PeopleFilters } from '../PeopleFilters';
```